lib/long_poll_transport.js

```diff
@@ -0,0 +1,156 @@
+import Transport from "./transport.js"
+
+export default class LongPollTransport extends Transport {
+  get messageSequenceHeader () {
+    return this._messageSequenceHeader
+  }
+
+  set messageSequenceHeader (str) {
+    this._messageSequenceHeader = (str || 'Pragma')
+  }
+
+  name() { return 'LongPoll'; }
+
+  // CORS is kinda supported in IE8+ except that its implementation cannot
+  // access "simple request" response headers. This means we don't yet have a
+  // plan to support IE<10 (when it gets a real XHR2 implementation). Sucks...
+  // $.browser.msie and parseInt($.browser.version) >= 8 # DEPRECATED
+  static ieSupported() { return (document.documentMode || 10) >= 8; }
```

Comment:
What is the lowest version of IE we want to target?

Reply:
If we decide to bump this, it should be in line with what Microsoft current supports.